terminology.md

```diff
@@ -0,0 +1,67 @@
+# Terminology
+
+## User
+
+Person identified with [WebID](https://www.w3.org/2005/Incubator/webid/spec/identity/).
+Assumed as trusted unless explicitly called **malicious user**.
+
+## WebID Profile
+
+Document resulting from dereferencing WebID [as specified](https://www.w3.org/2005/Incubator/webid/spec/identity/).
+No assumptions made about where it stays hosted.
+
+## Resource Server (RS) [RFC6749]
+
+The server hosting the protected resources, capable of accepting and responding to protected resource requests using access tokens.
+
+## Solid detail
```

Comment:
the subtitle `Solid detail` is a bit weird. I thought for a while reviewing this that this was a new term that you wanted to introduce. Given that there is only one sentence below perhaps there is no need to have a subsection. 
Can't think of a better title right now...